packages/bpk-component-bottom-sheet/src/BpkBottomSheet.module.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index.scss';
+
+$height: $bpk-line-height-base * 0.82;
+
+.bpk-container {
+  position: absolute;
+  z-index: 1;
```

Comment:
Accessibility have no concerns over using the z-index. https://skyscanner.slack.com/archives/C01GSKAPFB2/p1686560002652159?thread_ts=1686301427.090329&cid=C01GSKAPFB2